R/type-vctr.R

```diff
@@ -1,10 +1,17 @@
 #' vctr (vector) S3 class
 #'
+#' @description
 #' This abstract class provides a set of useful default methods that makes it
 #' considerably easier to get started with a new S3 vector class. See
 #' `vignette("s3-vector")` to learn how to use it to create your own S3
 #' vector classes.
 #'
+#' @details
+#' List vctrs are special cases. When created through `new_vctr()`, the
+#' resulting list vctr should always be recognized as a list by
+#' `vec_is_list()`. Because of this, the `inherit_base_type` argument is
+#' ignored for lists, and is always set to `TRUE`.
```

Comment:
Since the default value of `inherit_base_type` is not constant, it should be `NULL` instead of `FALSE`.

Reply:
Oops we talked about this, my bad